cypress/e2e/digital-lpa.cy.js

```diff
@@ -1,5 +1,54 @@
 describe("View a digital LPA", () => {
   beforeEach(() => {
```

Comment:
Does the mock need to be added before each test, or just before the suite? I think a `before()` property will only be called once for the suite.

Reply:
Looks like we [reset the mocks](https://github.com/ministryofjustice/opg-sirius-lpa-frontend/blob/main/cypress/support/index.js) after each test so `before()` makes only the first test work and others fail. I'm not entirely sure why we reset the mocks after each test, but i'm guessing its so that each test environment is refreshed so that tests are independent and isolated, and not dependant on each other to avoid a domino affect